storage/filestore/store_test.go

```diff
@@ -213,9 +216,134 @@ func TestDeleteWhileReading(t *testing.T) {
 		if info.IsDir() {
 			return nil
 		}
-		return errors.New("found file " + path)
+		return errs.New("found file %q", path)
 	})
 	if err != nil {
 		t.Fatal(err)
 	}
 }
+
+func writeABlob(ctx context.Context, t testing.TB, store *filestore.Store, blobRef storage.BlobRef, data []byte, formatVersion storage.FormatVersion) {
+	var (
+		blobWriter storage.BlobWriter
+		err        error
+	)
+	switch formatVersion {
+	case storage.FormatV0:
+		tStore := &filestore.StoreForTest{store}
+		blobWriter, err = tStore.CreateV0(ctx, blobRef)
+	case storage.FormatV1:
+		blobWriter, err = store.Create(ctx, blobRef, int64(len(data)))
+	default:
+		t.Fatalf("please teach me how to make a V%d blob", formatVersion)
+	}
+	require.NoError(t, err)
+	require.Equal(t, formatVersion, blobWriter.GetStorageFormatVersion())
+	_, err = blobWriter.Write(data)
+	require.NoError(t, err)
+	size, err := blobWriter.Size()
+	require.NoError(t, err)
+	assert.Equal(t, int64(len(data)), size)
+	err = blobWriter.Commit(ctx)
+	require.NoError(t, err)
+}
+
+func verifyBlobHandle(t testing.TB, reader storage.BlobReader, expectDataLen int, expectFormat storage.FormatVersion) {
+	assert.Equal(t, expectFormat, reader.GetStorageFormatVersion())
+	size, err := reader.Size()
+	require.NoError(t, err)
+	assert.Equal(t, int64(expectDataLen), size)
```

Comment:
I kind of feel like these extra messages only clutter the code and the output unless they're giving some information that the developer can't get otherwise (like what iteration of a loop it's on). In this case, I think that seeing "expectDataLen != size" should be a pretty good clue that this is a data length mismatch.

Reply:
> In this case, I think that seeing "expectDataLen != size" should be a pretty good clue that this is a data length mismatch.

Yes, that it would be perfect but unfortunately `assert.Equal` doesn't show the variable name so we will see something like

```
                Error:          Not equal: 
                                expected: 11
                                actual  : 10
```